jsonlines/jsonlines.py

```diff
@@ -325,7 +323,7 @@ def open(name, mode='r', **kwargs):
     """
     if mode not in {'r', 'w', 'a'}:
         raise ValueError("'mode' must be either 'r', 'w', or 'a'")
-    fp = io.open(name, mode=mode + 't', encoding='utf-8')
+    fp = builtins.open(name, mode=mode + 't', encoding='utf-8')
```

Comment:
`io.open()` is an alias for the built-in `open()` (see docs: https://docs.python.org/3/library/io.html#io.open):

```
In [1]: import io

In [2]: import builtins

In [3]: open is io.open
Out[3]: True

In [4]: open is builtins.open
Out[4]: True
```

but i agree with this change anyway since is more clearly signals the intent :+1:

Reply:
Without this change, it was calling the `open` function this line is actually in, which shadows the builtin, and then failing! So we need an explicit `builtins.open` here :)

Reply:
yeah i understand plain `open()` can't work, but `io.open()` solved that already? or do i miss something here? 🤔 (still agree with your change though 👍)

Reply:
Ah, it's one of the things auto-upgraded by pyupgrade :)

https://github.com/asottile/pyupgrade#open-alias